docs/topics/braintree.rst

```diff
@@ -287,6 +287,77 @@ Some information about the subscripton is stored in solitude.
     :query provider_id: the plan id for this subscription.
     :query seller_product: the primary key of the product.
 
+Sale
+----
+
+A sale is a one off payment to call the Braintree Transaction API. For more
+information see the `Braintree documentation <https://developers.braintreepayments.com/javascript+python/reference/request/transaction/sale>`_.
+
+Cannot be used for subscriptions.
+
+.. http:post:: /braintree/sale/
+
+    :<json amount: the amount of the transaction, within the maximum and minimum limits
+    :<json product_id: the product_id as defined by `payments-config <https://github.com/mozilla/payments-config/>`_
+    :<json nonce: (optional) product nonce, for use where no payment method is stored
```

Comment:
I don't think we have any more nonces, we have lots of tokens... its called a nonce here http://solitude.readthedocs.org/en/latest/topics/braintree.html#post--braintree-paymethod- and its the same value.